qcog_python_client/qcog.py

```diff
@@ -67,7 +67,7 @@ def get(self, *args: Any, **kwargs: Any) -> Coroutine | dict:
 SUCCESS_STATUS = (TrainingStatus.completed,)
 
 
-DEFAULT_QCOG_VERSION = "0.0.74"
+DEFAULT_QCOG_VERSION = "0.0.89"
```

Comment:
Are you bumping this manually or do we have a way to bump it automatically?

Reply:
The bump is manual. There is an ongoing discussion about how to change the way that specify the in-use qcog package from the client